src/main/java/io/github/terra121/control/fragments/terra/TerraWhereFragment.java

```diff
@@ -51,25 +53,29 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
 
             pos = e.getPositionVector();
             senderName = e.getName();
+            yaw = e.rotationYaw;
         }
 
         EarthGenerator terrain = (EarthGenerator) gen;
         GeographicProjection projection = terrain.projection;
 
 
         double[] result;
+        float azimuth;
         try {
             result = projection.toGeo(pos.x, pos.z);
+            azimuth = projection.azimuth(pos.x, pos.z, yaw);
         } catch (OutOfProjectionBoundsException e1) { //out of bounds, set to null to print error
             result = null;
+            azimuth = (Float) null;
```

Comment:
this will throw `java.lang.NullPointerException`.

Reply:
Do `azimuth = Float.NaN;`, and then test with `Float.isFinite(azimuth)`.